src/main/fc/boot.c

```diff
@@ -643,6 +646,13 @@ void init(void)
     afatfs_init();
 #endif
 
+    if (gyroConfig()->gyro_lpf > GYRO_LPF_256HZ && gyroConfig()->gyro_lpf < GYRO_LPF_NONE) {
```

Comment:
@ledvinap and @borisbstyle If you remember I was against removing the driver abstraction between the flight code and the driver code for the LPF.  At the time there were bigger fish to fry but now this rears it's ugly head and this is the result.